src/components/Pages/SetupPage/SetupPage.module.css

```diff
@@ -1,86 +1,190 @@
+  background-image: url('/src/assets/images/background.svg');
+  z-index: -1;
+}
+
 .left {
   flex: 1;
   padding: 1rem;
-  background-color: black;
   color: white;
   display: flex;
   justify-content: center;
   align-items: center;
 }
 
+.left :global(.message) {
+  color: white !important;
+  opacity: 70%;
+}
+
+.left :global(.ui.sub.header) > label {
+  color: white !important;
+  opacity: 70%;
+}
+
+.left input {
+  border-bottom: 2px solid rgba(255, 255, 255, 0.7) !important;
+}
+
+.left input::placeholder {
+  color: white !important;
+  opacity: 70%;
+}
+
 .leftInner {
   max-width: 500px;
   display: flex;
   flex-direction: column;
 }
 
 .logo {
-  width: 50px;
-  height: 50px;
+  width: 64px;
+  height: 64px;
 }
 
 .title {
-  margin-top: 2rem;
-  font-size: 2rem;
-  font-weight: 700;
+  font-size: 30px;
+  font-weight: 600;
+  line-height: 36px;
+  margin-top: 24px;
 }
 
 .subtitle {
-  margin-top: 1rem;
-  font-size: 1.8rem;
+  font-size: 30px;
+  font-weight: 400;
+  line-height: 36px;
 }
 
-.name {
-  margin-top: 8rem;
+.meetYourAvatar {
+  font-size: 18px;
+  font-weight: 600;
+  line-height: 22px;
+  margin-top: 24px;
 }
 
-.nameError {
-  color: var(--error);
-  display: inline-block;
-  margin-bottom: 1rem;
-  font-style: italic;
+.meetYourAvatarDescription {
+  font-size: 14px;
+  font-weight: 400;
+  line-height: 17px;
+  margin-top: 24px;
 }
 
-.email {
+.randomize {
+  margin-top: 16px;
 }
 
-.emailError {
-  color: var(--error);
-  font-style: italic;
+.randomize > :global(.button) {
+  display: flex;
+  align-items: center;
+  gap: 8px;
+}
+
+.continue {
+  width: 75%;
+  margin-top: 24px;
+}
+
+.logoSmall {
+  width: 40px;
+  height: 40px;
+}
+
+.back {
+  margin-top: 26px;
+  display: flex;
+  align-items: center;
+  gap: 8px;
+  cursor: pointer;
+}
+
+.back > span {
+  font-size: 14px;
+  font-weight: 600;
+  line-height: 18px;
+}
+
+.name {
+  margin-top: 24px;
+}
+
+.email {
+  margin-top: 24px;
 }
 
 .agree {
-  margin-top: 4rem;
+  margin-top: 24px;
   display: flex;
   align-items: center;
 }
 
-.checkbox {
+.agree > :global(.checkbox) {
   margin-right: 10px;
 }
 
+.agree :global(.ui.checkbox input[type='checkbox'].hidden ~ label:before) {
+  border: 2px solid rgba(255, 255, 255, 0.7);
+}
+
 .agree > a {
   color: white;
   font-weight: 700;
   text-decoration: underline;
 }
 
-.agreeError {
+.error {
   color: var(--error);
-  margin-top: 0.5rem;
   font-style: italic;
+  font-weight: 600;
+}
+
+.nameError {
+  display: inline-block;
+  margin-bottom: 1rem;
+}
+
+.emailError {
+  position: absolute;
+}
+
+.agreeError {
+  margin-top: 0.5rem;
+  opacity: 70%;
 }
 
 .jumpIn {
-  margin-top: 4rem;
+  margin-top: 24px;
 }
 
 .right {
   flex: 1;
+  width: 100%;
+  position: relative;
+  top: 10%;
+}
+
+.right :global(.WearablePreview) {
+  height: 80%;
+  position: relative;
+  z-index: 1px;
+
+}
+
+.platform {
+  display: block;
+  margin: auto;
+  transform: translateY(-110%);
+  width: 50vh;
 }
```

Comment:
We should use this style on other views as well for the platform + wearable preview as it makes them keep in place on all resolutions.